primefaces/src/main/java/org/primefaces/renderkit/MenuItemAwareRenderer.java

```diff
@@ -112,6 +112,10 @@ protected void encodeOnClick(FacesContext context, UIComponent source, MenuItem
                         : buildNonAjaxRequest(context, ((UIComponent) menuitem), form, ((UIComponent) menuitem).getClientId(context), true);
             }
 
+            if (isLink) {
+                // allow CTRL+CLICK link to open new tab
+                command = "if(event.ctrlKey||event.metaKey){return true};" + command;
```

Comment:
I'm wondering if we should do `PF.isMetaKey(event)` instead of `event.ctrlKey||event.metaKey`. If we would have a function, we could do OS checks. Something like
```js
return isMacOS ? event.metaKey : event.ctrlKey;
```
On macOS Control+Click will show the context menu by default.

Reply:
Great idea. Let me fix.